webpack/components/Content/Details/__tests__/__snapshots__/ContentDetails.test.js.snap

```diff
@@ -7,7 +7,7 @@ exports[`Content Details Info should render and contain appropriate components 1
     loadingText="Loading"
     timeout={300}
   >
-    <ForwardRef
+    <Uncontrolled(TabContainer)
```

Comment:
Is this intentional?  Seems Odd.

Reply:
@waldenraines I think this is related to a patternfly change.

Reply:
Foreman had similar changes once vendor v3 was released -- https://github.com/theforeman/foreman/commit/ae55b7eff2f2e653981781d3b73fc7d1361f1952